sub-projects/hub/src/app/components/admin/add-admin.jsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import axios from 'axios';
+import logger from 'utils/logger';
+
+import {
+  Button,
+  Form,
+  Input,
+  Header,
+  Grid,
+  Segment } from 'semantic-ui-react';
+
+
+class AddAdmin extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: '',
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+  handleChange({ target }) {
+    this.setState({
+      [target.name]: target.value,
+    });
+  }
+
+  handleSubmit() {
+    axios.post('http://127.0.0.1:3000/api/users', {
```

Comment:
Ideally we should take `http://127.0.0.1:3000` out of all the `axios` calls so it can be easily changed for production. The file I link to elsewhere in this review has some suggestions for how to do so